assets/src/edit-story/components/canvas/singleSelectionMovable.js

```diff
@@ -40,6 +41,7 @@ const ALL_HANDLES = [ 'n', 's', 'e', 'w', 'nw', 'ne', 'sw', 'se' ];
 function SingleSelectionMovable( {
 	selectedElement,
 	targetEl,
+	nodesById,
```

Comment:
We are directly accessing `useCanvas()` here anyway. So I'd rather get `nodesById` from there than passing it around. Could you also do the same for `MultiSelectionMovable`?